tokens/spacing.json

```diff
@@ -20,48 +20,68 @@
       "comment": "Grid column size"
     },
     {
-      "name": "SPACING_XX_SMALL",
-      "value": "0.444rem",
+      "name": "SPACING_1",
+      "value": "0.25rem",
       "comment": "For micro adjustments"
     },
     {
-      "name": "SPACING_X_SMALL",
-      "value": "0.667rem",
+      "name": "SPACING_2",
+      "value": "0.5rem",
       "comment": "For micro adjustments"
     },
     {
-      "name": "SPACING_SMALL",
+      "name": "SPACING_3",
+      "value": "0.75rem",
+      "comment": "For paddings in labels / inline stuff"
+    },
+    {
+      "name": "SPACING_4",
       "value": "1rem",
       "comment": "For paddings in labels / inline stuff"
     },
     {
-      "name": "SPACING_NORMAL",
+      "name": "SPACING_5",
+      "value": "1.25rem",
+      "comment": "For paddings in labels / inline stuff"
+    },
+    {
+      "name": "SPACING_6",
       "value": "1.5rem",
       "comment": "Regular padding for compact tables and such."
     },
     {
-      "name": "SPACING_MEDIUM",
+      "name": "SPACING_8",
+      "value": "2rem",
+      "comment": "Regular padding for compact tables and such."
+    },
+    {
+      "name": "SPACING_9",
       "value": "2.25rem",
       "comment": "For paddings in labels / inline stuff"
     },
     {
-      "name": "SPACING_LARGE",
-      "value": "3.375rem",
+      "name": "SPACING_12",
+      "value": "3rem",
+      "comment": "Padding for the inside of boxes"
+    },
+    {
+      "name": "SPACING_16",
+      "value": "4rem",
       "comment": "Padding for the inside of boxes"
     },
     {
-      "name": "SPACING_X_LARGE",
-      "value": "5.0625rem",
+      "name": "SPACING_20",
+      "value": "5rem",
       "comment": "Padding between elements and for boxes with larger font-sizes"
     },
     {
-      "name": "SPACING_XX_LARGE",
-      "value": "7.6rem",
+      "name": "SPACING_32",
```

Comment:
This naming convention where we jump from 8 to 9 to 12 to 16 will make the utilities hard to use. You'll constantly be thinking "what's the next spacing size we have available?". 

I'd like a range of [xs, sm, rg, md, lg, xl, xxl, xxxl] or just [1,2,3,4, ... 10].

Reply:
This naming convention ideally would be used every time we have a spacing scale.

Reply:
Ah yeah that makes sense. I currently used the factor of the 0.25 scale. But a follow up scale with numbers makes good sense also.

The reason I moved away from the xs, sm etc is that we have more steps now. So the naming becomes a bit akward.

Can we agree on going with 1, 2, 3 etc? (So just a follow up number, and not based on the actual factor)